### ISSUE TITLE
Accessibility of index page

### ISSUE THREAD

**Issue by salbeira** [decker-edu/decker#156]
This is a list of issues reported to us by the DoBuS of TU Dortmund about the accessibility of the index page. These issues are addressed already in the branch `dobus-feedback`. The improved version has been given to them for a second review and has received positive feedback.

The following issues were stated:

- Navigating through the links available on the index page is very tiring.
- The color mode can not be changed on a page view.

The issues were addressed in the following manner:

- It is recommended to use labeled links that contain the title of the presentation instead of listing the topic in a table and having nondescript labels to the slides like "HTML" or "PDF".
- You can enable a "modal link mode" where each link to a slide instead opens a dialog menu that allows you to pick what view the slides should be opened in (normal, handout, a11y mode). This also applies to links to slides that appear in the fuzzy search.

This can reduce the amount of inputs a non-sighted user has to perform to find the correct presentation.
